src/Payment.ts

```diff
@@ -317,17 +318,55 @@ export class Payment {
     )
   }
 
-  public async getTransferInformation(
+  public async getTransferInformationForTxHash(
```

Comment:
Why do we actually need this function, to look up the Transfers by hash, now that we have the event_id?